wntr/gis/network.py

```diff
@@ -171,7 +171,7 @@ def _extract_geodataframe(df, crs=None, links_as_points=False):
         df = df_links[df_links['link_type'] == 'Valve']
         self.valves = _extract_geodataframe(df, crs, valves_as_points) 
         
-    def _create_wn(self, append=None):
+    def _create_wn(self, append=None, index_col='index'):
```

Comment:
Adding `index_col` as an additional keyword argument is unnecessary here because the index name can be extracted from the dataframes.

Reply:
Thank you for your reply, help me to be able to use WNTR better